modules/virt-building-vm-containerdisk-image.adoc

```diff
@@ -0,0 +1,156 @@
+// Module included in the following assemblies:
+//
+// * virt//support/monitoring/virt-running-cluster-checkups.adoc              
+
+:_content-type: PROCEDURE                                             
+[id="virt-building-vm-containerdisk-image_{context}"]                                  
+= Building a container disk image for RHEL virtual machines
+
+You can build a custom RHEL 8 OS image in `qcow2` format and use it to create a container disk image. You can store the container disk image in a registry that is accessible from your cluster and specify the image location in the `spec.param.vmContainerDiskImage` attribute of the DPDK checkup config map. 
+
+Run the following procedure in the image builder virtual machine (VM) terminal to build a container disk image for RHEL 8.7 VMs with 2 CPU cores, 4 GiB RAM, and 20 GB of free space in the `/var` directory. The _image builder VM_ is a RHEL 8 VM that can be used to build custom RHEL images.
```

Comment:
```suggestion
In order to perform this procedure, you will have to create an image builder virtual machine.
The _image builder VM_ is a RHEL 8 VM that can be used to build custom RHEL images.
The image builder VM should run RHEL 8.7 and should have at least the following spec:
 2 CPU cores, 4 GiB RAM, and 20 GB of free space in the `/var` directory.
```

Reply:
Taking @orelmisan 's revision into consideration, I would suggest these slight revisions: 

In order to perform this procedure, you must create an image builder virtual machine (VM). The image builder VM is a {op-system-base} 8 VM that can be used to build custom {op-system-base} images.

I would move 

The image builder VM should run RHEL 8.7 and should have at least the following spec:
 2 CPU cores, 4 GiB RAM, and 20 GB of free space in the `/var` directory

to prerequisites